spark-java/pom.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.baeldung</groupId>
+  <artifactId>spark-java</artifactId>
+  <version>0.1.0-SNAPSHOT</version>
+  <name>spark-java</name>
+  <url>http://maven.apache.org</url>
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
```

Comment:
The project does not compile when I try to build locally with maven because the project uses Lambdas and maven is trying to compile to Java 1.5.
You have to set the version to 1.8 in the pom. Check some other project to see how they accomplish this.